config/options.go

```diff
@@ -301,6 +304,7 @@ var defaultOptions = Options{
 	RefreshDirectoryInterval:        10 * time.Minute,
 	RefreshDirectoryTimeout:         1 * time.Minute,
 	QPS:                             1.0,
+	ForwardAuthType:                 ProxyTypeNginx,
```

Comment:
I'm leaning towards not defaulting to any forward auth type here, failing hard, and telling the user this is now a required field when using forward auth and maybe even linking to the upgrade guide.

Reply:
Fixed.